cellengine/resources/gate.py

```diff
@@ -193,51 +96,177 @@ def create(
         x2: float,
         y1: float,
         y2: float,
-        label: List[str] = [],
+        label: List[float] = [],
         gid: str = None,
         locked: bool = False,
-        parent_population_id: str = None,
-        parent_population: str = None,
         tailored_per_file: bool = False,
         fcs_file_id: str = None,
         fcs_file: str = None,
-        create_population: bool = True,
     ) -> RectangleGate:
-        g = format_rectangle_gate(**get_args_as_kwargs(cls, locals()))
-        return cls(g)
+        """Formats a rectangle gate for posting to the CellEngine API.
 
+        Args:
+            experiment_id (str): The ID of the experiment.
+            x_channel (float): The name of the x channel to which the gate applies.
+            y_channel (float): The name of the y channel to which the gate applies.
+            name (str): The name of the gate x1 (float): The first x coordinate
+                (after the channel's scale has been applied).
+            x2 (float): The second x coordinate (after the channel's scale
+                has been applied).
+            y1 (float): The first y coordinate (after the channel's scale has
+                been applied).
+            y2 (float): The second y coordinate (after the channel's scale has
+                been applied).
+            label (float): Position of the label. Defaults to the midpoint of the gate.
```

Comment:
It's annoying to have to repeat the types here when they're already in the function signature, but the internet seems to say that's the way it has to be.

A few of these docstring types are out of sync. This one should be `List[float], optional`.